src/Host.cpp

```diff
@@ -1147,6 +1147,38 @@ bool Host::installPackage( const QString& fileName, int module )
     }
     // reorder permanent and temporary triggers: perm first, temp second
     mTriggerUnit.reorderTriggersAfterPackageImport();
+
+    // This event was inspired by "Zoilder" in a Mudlet derivative in 2015
```

Comment:
Well it was an acknowledgement of the source material which we cannot include in the Copyrights at the top because we do not have the full details...

Wasn't aware of the other requests but that is good to know - just thought it would be wanted at least by the originator and could encourage them to come to the source in the future.